mypy.ini

```diff
@@ -30,3 +31,9 @@ ignore_missing_imports = True
 
 [mypy-pandas.*]
 ignore_missing_imports = True
+
+[mypy-django.*]
```

Comment:
I think django has a stub library. You could just install that.

Reply:
Hmm, yes most of my `mypy-pre-commit` errors were related to `django-stub`. It was just bleedfixing solution. I will add it to install before running `mypy-pre-commit` in the next PR